src/main/webapp/app/lecture/lecture-attachments.component.html

```diff
@@ -175,30 +177,25 @@ <h4 jhiTranslate="artemisApp.lecture.attachments.editAttachment"></h4>
                                     }
                                 </div>
                                 <div class="form-group">
-                                    <jhi-date-time-picker
-                                        labelName="{{ 'artemisApp.lecture.attachments.releaseDate' | artemisTranslate }}"
-                                        [(ngModel)]="attachmentToBeCreated.releaseDate"
-                                        name="endDate"
-                                    />
+                                    <jhi-date-time-picker labelName="{{ 'artemisApp.lecture.attachments.releaseDate' | artemisTranslate }}" formControlName="releaseDate" />
                                 </div>
-                                @if (attachmentToBeCreated.id) {
+                                @if (attachmentToBeUpdatedOrCreated()!.id) {
```

Comment:
optional: is this ! necessary?

Reply:
It should not be, as we check for undefined in line 127, but otherwise there will be warnings in IntelliJ - the language service does not seem to fully support signals yet and I think makes the code less maintainable if there are warnings everywhere instead of having non null assertions if there is a check beforehand.